Tests/SmokeDynamoDBTests/DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests.swift

```diff
@@ -95,6 +131,35 @@ class DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests: XCTestC
         XCTAssertEqual("secondlyX2", secondRetrievedItem.rowValue.secondly)
     }
     
+    func testUpdateItemConditionallyAtKeyWithAcceptableConcurrencyWithAsyncProvider() {
+        let wrappedTable = InMemoryDynamoDBCompositePrimaryKeyTable(eventLoop: eventLoop)
+        let table = SimulateConcurrencyDynamoDBCompositePrimaryKeyTable(wrappedDynamoDBTable: wrappedTable, eventLoop: eventLoop,
+                                                     simulateConcurrencyModifications: 5,
+                                                     simulateOnInsertItem: false)
+        
+        let key = StandardCompositePrimaryKey(partitionKey: "partitionId",
```

Comment:
Why are we not using `testPrimaryItemProvider(nil)` here like in the other tests?

Reply:
`testPrimaryItemProvider` is just used in the DynamoDBCompositePrimaryKeyTableHistoricalItemExtensionsTests.swift. This test looks like it is consistent with the other tests in this file.